www/js/diary/infinite_scroll_place_item.js

```diff
@@ -0,0 +1,47 @@
+
+/**
+ * A directive to display each place within the diary view.
+ */
+
+angular.module('emission.main.diary.infscrollplaceitem',
+    ['emission.main.diary.infscrolllist',
+        'emission.survey.multilabel.services',
+        'emission.main.diary.infscrolldetail',
+        'ng-walkthrough',
+        'nvd3',
+        'emission.plugin.kvstore',
+        'emission.services',
+        'emission.config.imperial',
+        'emission.config.dynamic',
+        'emission.plugin.logger',
+        'emission.stats.clientstats',
+        'emission.survey.enketo.add-note-button',])
+
+  .directive("infiniteScrollPlaceItem", function () {
+    return {
+      restrict: 'E',
+      scope: {
+        trip: '=',
+        config: '=',
+        surveys: '=',
+      },
+      controller: 'PlaceItemCtrl',
+      templateUrl: 'templates/diary/place_list_item.html'
+    };
+  })
+
+  .controller("PlaceItemCtrl", function ($scope, $state, DynamicConfig) {
+    console.log("Place Item Controller called");
+    console.log('config is ', $scope.config);
+
+    $scope.timeBounds = () => {
+      let obj = {
+        isPlace: true,
+        start_fmt_time: $scope.trip?.end_fmt_time,
+        end_fmt_time: $scope.trip?.nextTrip?.start_fmt_time
+      }
+      if(!obj.end_fmt_time) obj.end_fmt_time = obj.start_fmt_time;
```

Comment:
@sebastianbarry 

this is what prefills in the time use survey for that most recent place. We thought this was better than leaving it blank.
However, a better solution might be to grab the current time (because the last place is presumably your current location)

Reply:
Changed this to prefill with the current time as a default option (i.e. the last place in the list):

https://github.com/e-mission/e-mission-phone/pull/917/commits/82910086ea6ce874ff571abd79c5d049f38ab456